src/utils.rs

```diff
@@ -0,0 +1,54 @@
+use macroquad::{
+    experimental::collections::storage, experimental::scene, experimental::scene::RefMut,
+    prelude::Vec2, rand::gen_range,
+};
+
+use crate::{nodes::Player, Circle, Resources};
+use std::f32;
+
+/// Creates explosion FX and kills players within `radius`
+pub fn explode(position: Vec2, radius: f32) {
```

Comment:
"explode" is still used in exactly one place. I still think that it belong to, now, "explosive.rs"

Reply:
Moved